README.md

```diff
@@ -1158,6 +1159,7 @@ See **[Ticketing](#ticketing)**
 - [JS Bin](http://jsbin.com/) - Open source collaborative web development debugging tool. ([Source Code](https://github.com/jsbin/jsbin)) `MIT` `Nodejs`
 - [Judge0 API](https://api.judge0.com) - Open source API to compile and run source code. ([Source Code](https://github.com/judge0/api)) `GPL-3.0` `Ruby`
 - [Koding](http://www.koding.com/) - The simplest way to manage your entire Dev Infrastructure. ([Source Code](https://github.com/koding/koding)) `Apache-2.0` `Nodejs`
+- [Lepton](https://github.com/hackjutsu/Lepton) - Lepton is a lean snippet manager based on GitHub Gist. `MIT` `React`
```

Comment:
- Requires a third-party service (Github) and must be marked `⚠` (see https://github.com/Kickball/awesome-selfhosted#list-of-licenses, sorry this requirement was removed from contributing guidelines in error)
- `Nodejs`
- Edit: Just saw it is a desktop/electron app, so it does not belong to the list. Please remove it.